keras_hub/src/models/causal_lm.py

```diff
@@ -339,6 +339,33 @@ def preprocess(x):
         def generate(x):
             return generate_function(x, stop_token_ids=stop_token_ids)
 
+        def strip_prompt_fn(x, prompt):
+            # This function removes the prompt from the generated
+            # response, in a batch-friendly fashion.
```

Comment:
It'd be great to turn this comment into a docstring.

Reply:
I meant using `"""`  for this comment instead of `#` so it'll show up as a docstring.